src/virt-install

```diff
@@ -252,6 +262,14 @@ try:
     # to ensure we fully control the code.
     cleanup_argv = ['/usr/lib/coreos-assembler/gf-anaconda-cleanup', args.dest]
     run_sync_verbose(cleanup_argv)
+
+    # This one is part of the "API" we provide to config repositories.  At least
+    # RHEL CoreOS will use it to change the OS to appear to have been provisioned
+    # from an oscontainer.
+    config_image_post = image_conf.get("postprocess-script")
```

Comment:
does this postprocess apply to all images output of the pipeline (qemu, metal-bios, metal-uefi)?

Reply:
Yep.